DbConnection/src/main/java/com/solvd/laba/db/dao/MySQLDAO/MySQLCustomerDAO.java

```diff
@@ -0,0 +1,104 @@
+    private final static String INSERT_VALUES = "INSERT INTO customer(first_name, last_name, phone_number, email, age) VALUES (?, ?, ?, ?, ?)";
+    private final static String UPDATE_VALUE = "UPDATE customer SET first_name=?, last_name=?, phone_number=?, email=?, age=? WHERE id=?";
+    private final static String DELETE_VALUE = "DELETE FROM customer WHERE id=?";
+
+    @Override
+    public Customer get(int id) {
+        logger.info("Finding record of ID: " + id + ".....");
+        try (PreparedStatement preparedStatement = getConnection().prepareStatement(GET_BY_ID)) {
+            preparedStatement.setInt(1, id);
+            ResultSet result = preparedStatement.executeQuery();
+            result.next();
+            String firstName = result.getString("first_name");
+            String lastName = result.getString("last_name");
+            String phoneNumber = result.getString("phone_number");
+            String email = result.getString("email");
+            int age = result.getInt("age");
+            return new Customer(firstName, lastName, email, phoneNumber, age);
+        } catch (SQLException se) {
+            logger.info("No record found. Invalid ID");
+            return null;
+        }
+    }
+
+    @Override
+    public ArrayList<Customer> get() {
+        logger.info("Displaying all the rows from customer table");
+        ArrayList<Customer> customers = new ArrayList<>();
+        try (PreparedStatement preparedStatement = getConnection().prepareStatement(GET_ALL)) {
+            ResultSet result = preparedStatement.executeQuery();
+            while (result.next()) {
+                int id = result.getInt("id");
+                String firstName = result.getString("first_name");
+                String lastName = result.getString("last_name");
+                String phoneNumber = result.getString("phone_number");
+                String email = result.getString("email");
+                int age = result.getInt("age");
+                Customer customer = new Customer(id, firstName, lastName, email, phoneNumber, age);
+                customers.add(customer);
+            }
+            logger.info(customers);
+            return customers;
+        } catch (SQLException e) {
+            logger.info("Error:"+e.getMessage());
+        }
+        return null;
+    }
+
+    @Override
+    public void add(Customer row) {
+        try (PreparedStatement preparedStatement = getConnection().prepareStatement(INSERT_VALUES)) {
+            preparedStatement.setString(1, row.getFirstName());
+            preparedStatement.setString(2, row.getLastName());
+            preparedStatement.setString(3, row.getPhoneNumber());
+            preparedStatement.setString(4, row.getEmail());
+            preparedStatement.setInt(5, row.getAge());
+            preparedStatement.executeUpdate();
+            logger.info("Insertion complete");
+        } catch (SQLException e) {
+            logger.info("Error:"+e.getMessage());
```

Comment:
Maybe make all the error Texts the same in all DAO classes

Reply:
Done